Bento/Views/BentoReusableView.swift

```diff
@@ -66,3 +72,13 @@ extension BentoReusableView where Self: UIView {
         }
     }
 }
+
+fileprivate extension UIView {
+    func enumerateAllViews(_ action: (UIView) -> Void) {
```

Comment:
`enumerateAllViews` is not very clear about what this does 😕  Perhaps `forViewAndAllSubviews` would make it clearer what is actually happening?